builtin/logical/pki/cert_util.go

```diff
@@ -1183,3 +1186,66 @@ NameCheck:
 
 	return fmt.Errorf("name %q disallowed by CA's permitted DNS domains", badName)
 }
+
+func convertRespToPKCS8(resp *logical.Response) error {
+	privRaw, ok := resp.Data["private_key"]
+	if !ok {
+		return nil
+	}
+	priv, ok := privRaw.(string)
+	if !ok {
+		return fmt.Errorf("error converting response to pkcs8: could not parse original value as string")
+	}
+
+	privKeyTypeRaw, ok := resp.Data["private_key_type"]
+	if !ok {
+		return fmt.Errorf("error converting response to pkcs8: %q not found in response", "private_key_type")
```

Comment:
Can we not use `%q` here since its not taking any variable as input? Or was there a reason for doing it this way?

Reply:
I did it this way instead of `\"`. Just felt nicer than escapes.

Reply:
Ah, I see. 👍